WooCommerce/UITestsFoundation/Screens/Orders/CustomerNoteScreen.swift

```diff
@@ -31,6 +31,13 @@ public final class CustomerNoteScreen: ScreenObject {
         return self
     }
 
+    @discardableResult
+    public func verifyNoteAdded(_ text: String) throws -> Self {
+        XCTAssertTrue(app.keyboards.firstMatch.exists)
+        XCTAssertEqual(noteTextEditor.value as? String, text)
+        return self
+    }
```

Comment:
Just thinking aloud: this checks if customer note exists, but it does not know if it's because it was just added, or it was there for a long time. It can as well be called `verifyNoteExists` (but then, I suppose, keyboard should not be present 🤔).

After some thinking, though, I guess the combination of text + keyboard presence are signs of a just added note, so keeping it as is, is also fine.